subsys/bluetooth/controller/ll_sw/ull_adv.c

```diff
@@ -704,7 +705,12 @@ u8_t ll_adv_enable(u8_t enable)
 		lll_hdr_init(&conn->lll, conn);
 
 		/* wait for stable clocks */
-		lll_clock_wait();
+		err = lll_clock_wait();
```

Comment:
In the current implementation(s) lll_clock_wait always returns 0.
If this is for preparing for an implementation where we actually can check for stable clock I propose saying this in a comment

Reply:
Updated with timeout in k_sem_take and returns the err value.